pkg/services/featuremgmt/openfeature.go

```diff
@@ -93,14 +93,9 @@ func InitOpenFeatureWithCfg(cfg *setting.Cfg) error {
 	})
 }
 
-func createProvider(
-	providerType string,
-	u *url.URL,
-	staticFlags map[string]bool,
-	httpClient *http.Client,
-) (openfeature.FeatureProvider, error) {
+func createProvider(providerType string, u *url.URL, staticFlags map[string]setting.FeatureToggle, httpClient *http.Client) (openfeature.FeatureProvider, error) {
 	if providerType != setting.GOFFProviderType {
-		return newStaticProvider(staticFlags)
+		return newStaticProvider(staticFlags, standardFeatureFlags)
```

Comment:
Asking out of curiosity: why did you decide to pass standard feature flags here vs how it was before? Thanks!